app/renderer/components/urlBarIcon.js

```diff
@@ -11,6 +11,19 @@ const dndData = require('../../../js/dndData')
 const {isSourceAboutUrl} = require('../../../js/lib/appUrlUtil')
 const searchIconSize = 16
 
+const getIconCssClass = (ctx) => {
+  if (ctx.isSearch) {
```

Comment:
I'm not quite sure how it does work but do you mean `ctx.isSearch && !ctx.isAboutPage` is not necessary? I'm thinking the case where you try to search something on the URL bar on about:preferences.